### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by tOPMOM2011** [derrod/legendary#677]
so i downloaded fortnite without a problem but now there is a problem. Fortnite needs anti cheat to run and i don't have admin sooooo idk how to do this could you help. or can you tell me were legendary downloads fortnite so i can put the easy cheat dll in there

**Comment by CommandMC** [comment]
If you don't have full access to the computer you're using to play games, chances are you're not supposed to play games on it

Fortnite will not run without administrative permissions. If you don't have those, you're not playing the game

**Comment by tOPMOM2011** [comment]
i have the dll file for easy anti cheat

**Comment by tOPMOM2011** [comment]
won't that work

**Comment by CommandMC** [comment]
No, it won't. Again, (1) Fortnite requires administrative permissions and (2) if you don't have those permissions, you're most likely on a computer not owned by yourself, which is not _supposed to_ run games in the first place.
If this is the case, we cannot help you bypass those restrictions. If it is not (and you believe you should have administrative permissions), contact the system administrator and ask them about it.

**Comment by tOPMOM2011** [comment]
thank you have a good day
